trigger-openqa_in_openqa

```diff
@@ -39,6 +39,7 @@ main() {
     # and use for submit requests
     if [[ $full_run ]]; then
         staging_project=$src_project
+        staging_group_id=$group_id
```

Comment:
just to make sure: `$full_run` means it should run [openQA Full](https://openqa.opensuse.org/group_overview/133) or [Factory Submissions](https://openqa.opensuse.org/group_overview/135)?

Reply:
It should run [openQA Full](https://openqa.opensuse.org/group_overview/133).